README.md

```diff
@@ -8,4 +8,14 @@ The script receive two parameters:
 the list should be separated by commas
 
 Example:
-./pair-swap.sh -a ania,tom,john -f megan,james,sarah
+
+`$ ./pair-swap.sh -a ania,tom,john -f megan,james,sarah`
+
+
+On OXS you could need install `coreutils`:
+
+`$ brew install coreutils`
+
+If running with ZSH (Z shell):
```

Comment:
if you have bash installed it will work as it's declared to use bash on the shebang.
I use fish as my default shell here and it works fine because I have bash installed.
Maybe we can change this to something like "It requires bash and coreutils to run"?